### DISCUSSION TITLE
Kick.com integration

### DISCUSSION THREAD

**Discussion by sinnx3** [Chatterino/chatterino2#4676] [Ideas] [answered]
With the twitch exodus underway, I feel like chatterino would be a great asset for the kick streaming platform.

**Comment by Felanbird** [comment, marked as answer]
Chatterino would indeed be a great asset for kick, but we will not be adding kick support.

2023-06-27 edit: Some kick dev (un-sure if 3rd party) has announced he plans to release their own client.

**Reply by rostby** [reply]
where that client at it's been a year

**Reply by iProdigy** [reply]
don't ask us; kick still hasn't released a public api for a third-party dev to be able to build a stable chat client (and we have enough self-respect to not be willing to write that code)
